github/src/main/kotlin/gropius/sync/github/Outgoing.kt

```diff
@@ -289,6 +291,28 @@ class Outgoing(
         }
     }
 
+    /**
+     * Mutate an RemovedLabelEvent upto GitHub
```

Comment:
this doc sounds wrong

Reply:
9a26f69d17142b37ebdf4b1aeee8efbaa0d61920